feature-libs/organization/account-summary/root/account-summary-root.module.ts

```diff
@@ -0,0 +1,50 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import {
+  AuthGuard,
+  CmsConfig,
+  provideDefaultConfig,
+  provideDefaultConfigFactory,
+} from '@spartacus/core';
+import { AdminGuard } from '@spartacus/organization/administration/core';
```

Comment:
This import will split `administration` bundle, and make the `administration core` part load eagerly. Maybe need to move `AdminGuard` to `@spartacus/organization/administration/root`.

Reply:
I tried this, but it will cause alot of errors and is out of scope.

Reply:
Can move AuthGuard, AdminGuard to component level. e.g.
```
export const accountSummaryHeaderCmsConfig: CmsConfig = {
  cmsComponents: {
    AccountSummaryHeaderComponent: {
      component: AccountSummaryHeaderComponent,
      guards: [AuthGuard, AdminGuard],
    },
  },
};
```
also add guards to `AccountSummaryDocumentComponent`.

Reply:
done